specifyweb/specify/migration_utils/update_schema_config.py

```diff
@@ -211,36 +238,52 @@ def update_table_field_schema_config_with_defaults(
             f"Field does not exist in latest state of the datamodel, skipping Schema Config entry for: {table_name} -> {field_name}"
         )
         return
+    
+    # Apply defaults if provided
+    field_name_str = field.name
```

Comment:
```suggestion
    field_name_str = camel_to_spaced_title_case(field.name)
```

I should have left a comment when I worked with it, but the following was actually intentional and not a bug/typo in the original code 😉 

https://github.com/specify/specify7/blob/e7123b6feab07e21d794bd1a2890c601de56b3e1/specifyweb/specify/migration_utils/update_schema_config.py#L219

https://github.com/specify/specify7/blob/e7123b6feab07e21d794bd1a2890c601de56b3e1/specifyweb/specify/migration_utils/update_schema_config.py#L234-L235

Currently, field labels (captions) will be created with their datamodel names. Take a look at some fields which are not present in the defaults from #7444: 

https://github.com/user-attachments/assets/0a845938-91bb-4713-a980-845be7c39803